canvas_modules/common-canvas/package.json

```diff
@@ -92,29 +87,27 @@
     "jest": "26.4.2",
     "jest-fetch-mock": "3.0.3",
     "jest-localstorage-mock": "2.4.3",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-intl": "5.8.2",
-    "react-redux": "7.2.1",
-    "react-test-renderer": "16.13.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-intl": "6.6.2",
+    "react-redux": "7.2.8",
+    "react-test-renderer": "18.2.0",
     "reactable": "1.1.0",
     "rollup": "2.60.1",
     "rollup-plugin-auto-external": "2.0.0",
     "rollup-plugin-scss": "3.0.0",
     "rollup-plugin-terser": "7.0.2",
     "rollup-plugin-visualizer": "5.5.2",
-    "sass": "1.43.5",
+    "sass": "^1.71.1",
     "sinon": "9.0.3",
-    "stylelint": "13.13.1"
+    "stylelint": "13.13.1",
+    "url": "0.11.3"
```

Comment:
How come url was added?

Reply:
After upgrade to Webpack 5.x, [jsonschema validator](https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/schema-validator/properties-schema-validator.js#L27) started failing with error -
```
scan.js:28 Uncaught TypeError: urilib.resolve is not a function
```
App used to crash with the above error while opening properties, palette, canvas from JSON files. Error was fixed by adding `url` package.